keyboards/trainpad/trainpad.h

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2023 Ananya Kirti
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT(k000, k001, k002, k003) {{ k000,  k001,  k002,  k003 }}
```

Comment:
As per `config.h`, this should be defined on `info.json` and then this file deleted

Reply:
is it necessary to move it to info.json? can you suggest changes? I'll commit them

Reply:
committed the changes! Kindly let me know if they are fine!